docs/content/learn/tutorials/general/releases.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Testing New Releases
+heading: Testing New OSCAL Releases
+Description: A tutorial that 
+suppresstopiclist: true
+toc:
+  enabled: true
+alias:
+- /learn/tutorials/releases/
+---
+
+This tutorial provides common ways community members can test a new release.
+
+Before reading this tutorial, you should have a basic familiarity with:
+
+- The OSCAL models and their [overall structure](/concepts/layer/overview/).
+- [XML](https://www.w3.org/standards/xml/core), [JSON](https://www.json.org/), or [YAML](https://yaml.org/spec/) formats.
+- The basics of [version control with Git and GitHub](https://docs.github.com/en/get-started/using-git/about-git).
+
+## The OSCAL release model
+
+OSCAL releases roughly follow a [semantic versioning](https://semver.org/) convention of `MAJOR`.`MINOR`.`PATCH` (for more details, see [the development roadmap](/contribute/roadmap/)).
+
+For the most part, minor and patch releases have a backwards-compatibility guarantee with exceptions announced on the [GitHub discussion page](https://github.com/usnistgov/OSCAL/discussions).
+
+### How do I provide feedback on a new release?
+
+The OSCAL team encourages community members to submit feedback and ask questions as [GitHub issues](https://github.com/usnistgov/OSCAL/issues) or to the [release's accompanying GitHub discussion](https://github.com/usnistgov/OSCAL/discussions/categories/releases).
+
+## Testing a new release
+
+All releases are published to [OSCAL GitHub repository](https://github.com/usnistgov/OSCAL) in the [releases section](https://github.com/usnistgov/OSCAL/releases).
+
+Releases are [tagged in Git](https://git-scm.com/book/en/v2/Git-Basics-Tagging) automatically with a "v" prefix.
+For example, a release for version 1.0.0 would be tagged `v1.0.0`.
+
+To test a specific release, [clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) and checkout the release's associated tag.
```

Comment:
Minor, non-blocking: add some whitespace like other paragraphs?

```suggestion
To test a specific release, [clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) and checkout the release's associated tag.

```

Reply:
I'm dismissing whitespace related feedback here, I tend to follow the convention of [one sentence per line](https://nick.groenen.me/notes/one-sentence-per-line/) for markdown documents. This means that a paragraph will have each sentence on its own line.

Reply:
I'm cool with that, I just have trouble quickly skimming and knowing if, in GitHub, if the newline is just flow wrapping of a phrase and is intentionally a new paragraph without an empty line between paragraphs in such Markdown files. I am fine without, hence I allow you to ignore it. 😄 Thanks again!